backend/api/migrations/0003_manual_view.py

```diff
@@ -141,55 +56,173 @@ def create_materialize_proc(apps, schema_editor):
-            COALESCE(t.sum_ffp_units, 0) AS ffp_units,
-            COALESCE(t.sum_plt_units, 0) AS plt_units,
-            COALESCE(t.sum_cryo_units, 0) AS cryo_units,
-            COALESCE(t.sum_whole_units, 0) AS whole_units,
-            COALESCE(t.sum_cell_saver_ml, 0) AS cell_saver_ml,
-            v.clinical_los AS los,
-            CASE WHEN v.pat_expired_f = 'Y' THEN TRUE ELSE FALSE END AS death,
-            CASE WHEN v.total_vent_mins > 1440 THEN TRUE ELSE FALSE END AS vent,
-            CASE WHEN bc.stroke = 1 THEN TRUE ELSE FALSE END AS stroke,
-            CASE WHEN bc.ecmo = 1 THEN TRUE ELSE FALSE END AS ecmo,
-            CASE WHEN m.has_b12 = 1 THEN TRUE ELSE FALSE END AS b12,
-            CASE WHEN m.has_iron = 1 THEN TRUE ELSE FALSE END AS iron,
-            CASE WHEN m.has_antifibrinolytic = 1 THEN TRUE ELSE FALSE END AS antifibrinolytic,
-            COALESCE(ga.rbc_adherent, 0) AS rbc_adherent,
-            COALESCE(ga.ffp_adherent, 0) AS ffp_adherent,
-            COALESCE(ga.plt_adherent, 0) AS plt_adherent,
-            COALESCE(ga.cryo_adherent, 0) AS cryo_adherent
-        FROM Visit v
+            
+            COALESCE(pt.sum_rbc_units, 0),
+            COALESCE(pt.sum_ffp_units, 0),
+            COALESCE(pt.sum_plt_units, 0),
+            COALESCE(pt.sum_cryo_units, 0),
+            COALESCE(pt.sum_whole_units, 0),
+            COALESCE(pt.sum_cell_saver_ml, 0),
+
+            -- Visit outcomes assigned only to admitting attending
+            CASE WHEN ap.attend_prov_line = 0 THEN v.clinical_los ELSE NULL END AS los,
+            CASE WHEN ap.attend_prov_line = 0 THEN (CASE WHEN v.pat_expired_f = 'Y' THEN TRUE ELSE FALSE END) ELSE NULL END AS death,
+            CASE WHEN ap.attend_prov_line = 0 THEN (CASE WHEN v.total_vent_mins > 1440 THEN TRUE ELSE FALSE END) ELSE NULL END AS vent,
+            CASE WHEN ap.attend_prov_line = 0 THEN (CASE WHEN bc.stroke = 1 THEN TRUE ELSE FALSE END) ELSE NULL END AS stroke,
+            CASE WHEN ap.attend_prov_line = 0 THEN (CASE WHEN bc.ecmo = 1 THEN TRUE ELSE FALSE END) ELSE NULL END AS ecmo,
+
+            -- Medications per provider
+            CASE WHEN pm.has_b12 = 1 THEN TRUE ELSE FALSE END AS b12,
+            CASE WHEN pm.has_iron = 1 THEN TRUE ELSE FALSE END AS iron,
+            CASE WHEN pm.has_antifibrinolytic = 1 THEN TRUE ELSE FALSE END AS antifibrinolytic,
+
+            -- Adherence per provider
+            COALESCE(pt.rbc_adherent, 0),
+            COALESCE(pt.ffp_adherent, 0),
+            COALESCE(pt.plt_adherent, 0),
+            COALESCE(pt.cryo_adherent, 0),
+
+            ap.prov_name,
+            ap.prov_id,
+            ap.attend_prov_line,
+            CASE WHEN ap.attend_prov_line = 0 THEN TRUE ELSE FALSE END AS is_admitting_attending
+
+        FROM (
+            SELECT 
+                visit_no, 
+                prov_id, 
+                MAX(prov_name) as prov_name, 
+                MIN(attend_prov_line) as attend_prov_line
```

Comment:
You're only giving one provider/visit row in this table. I was expecting that we would have something like

```
visit_no,provider,provider_line,rbc_units,...
1,Jack,0,1
1,Luke,1,1
1,Jack,2,1
```

NOTE: Jack is there twice with 2 provider line numbers, 0 and 2. This would give us the provenance of handoffs throughout the visit

Then we could group by visit_no and provider to get the per visit stats per provider. LMK what you think